web/source/dom/domManager.ts

```diff
@@ -907,6 +922,30 @@ namespace com.keyman.dom {
       }
     }.bind(this);
 
+    _InputModeObserverCore = function(this: DOMManager, mutations: MutationRecord[]) {
+      const keyman = com.keyman.singleton;
+      // Prevent infinite recursion from any changes / updates made within the observation handler.
+      this.disableInputModeObserver();
+      try {
+        for(const mutation of mutations) {
+          const target = mutation.target as HTMLElement;
+          if(!this.isAttached(target)) {
+            continue;
+          }
+
+          const newValue = target.inputMode;
+
+          target._kmwAttachment.inputMode = newValue;
```

Comment:
```suggestion
          target._kmwAttachment.inputMode = target.inputMode ?? 'text';
```

Don't really need the extra `const` here. Also, to keep it consistent with the other location where we set `_kmwAttachment.inputMode`?